client/http/client.go

```diff
@@ -446,6 +467,48 @@ func (c *client) DeletePlacementRule(ctx context.Context, group, id string) erro
 		http.MethodDelete, http.NoBody, nil)
 }
 
+// GetAllPlacementRuleGroups gets all placement rule groups.
+func (c *client) GetAllPlacementRuleGroups(ctx context.Context) ([]*RuleGroup, error) {
+	var ruleGroups []*RuleGroup
+	err := c.requestWithRetry(ctx,
+		"GetAllPlacementRuleGroups", placementRuleGroups,
+		http.MethodGet, nil, &ruleGroups)
```

Comment:
maybe need to sync with this change
https://github.com/tikv/pd/pull/7428/files#diff-138f7d23a1171f829bb57a7edcd66e8ac0ceb81f1cb2f28e497284c732aaad22L339

Reply:
Done. PTAL.